Misha/Files/quiz_maker/quiz_maker_main.py

```diff
@@ -0,0 +1,101 @@
+import time
+import random
+import os
+
+
+def get_questions(path):
+    if not os.path.exists(path):
+        raise FileNotFoundError('File not found')
+    else:
+        f = open(path, 'r', encoding='utf-8')
+        text = ''
+        for line in f:
+            text += line
+        f.close()
```

Comment:
Please close file in final block or use with-resource statement

Reply:
why cant i close it after finishin using it?